### PR TITLE
Created "progress-list-item" component

### PR DESCRIPTION
**Pull request by NFSS10**
| - | - |
| --- | --- |
| Issue | - |
| Dependencies |  • https://github.com/ripe-tech/ripe-components-vue/pull/272 <br>• https://github.com/ripe-tech/ripe-components-vue/pull/276  |
| Decisions | Created `progress-list-item` component that is used in `progress-list` component |
| Animated GIF | progress-list-item |

### CODE REVIEW

File: vue/components/ui/molecules/progress-list-item/progress-list-item.vue [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+            type: String,
+            default: null
+        },
+        actionIcon: {
+            type: String,
+            default: null
+        }
+    },
+    methods: {
+        onActionButtonClick(event) {
```
Comment (BeeMargarida) [inline vue/components/ui/molecules/progress-list-item/progress-list-item.vue]: event is not being used

**Comment by NFSS10** [comment]
Reassigning to me again, so that I can make that animation that we discussed earlier today.

**Comment by gcandal** [comment]
@NFSS10 why is this draft?

**Comment by NFSS10** [comment]
> @NFSS10 why is this draft?

Waiting for dependencies to be merged

**Comment by gcandal** [comment]
> > @NFSS10 why is this draft?
> 
> Waiting for dependencies to be merged

Weren't they merged already? Unless you missed some dependency in the list. In any case, please merge against master.

**Pull request merged**